lib/rubocop/cop/style/invertible_unless_condition.rb

```diff
@@ -99,23 +99,23 @@ def preferred_condition(node)
           end
         end
 
-        def preferred_send_condition(node)
-          receiver_source = node.receiver.source
+        def preferred_send_condition(node) # rubocop:disable Metrics/CyclomaticComplexity
+          receiver_source = node.receiver&.source
           return receiver_source if node.method?(:!)
 
+          receive = receiver_source ? "#{receiver_source}." : '' # receiver may be implicit (self)
```

Comment:
I chose to do this instead of repeating the conditional

```ruby
if receiver_source
  # ...
else
  # ...
end
```

in three places, since it's always the same logic.

I also considered `receipt` as the variable name. Not sure.

Reply:
Any reason why you didn't go with the name `receiver`?

Reply:
I wasn't sure if that would be confusing because we would then have `receiver_source` and `receiver`, but `receiver` would be `receiver_source + '.'` or `nil`, not the "receiver" node, as one might expect.

I suppose another option could be `dotted_receiver` or `dotted_receiver_source`.

Reply:
Yeah. I did switch it to `dotted_receiver_source` though, since that's clearer than `receive`.